thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -72,6 +73,23 @@ export default Component.extend({
     }
   ),
 
+  /**
+   * Metric urn for anomaly topic metric
+   * @type {string}
+   */
+  metricUrn: computed(
+    'anomalyUrns',
+    function () {
+      const { anomalyUrns } = getProperties(this, 'anomalyUrns');
```

Comment:
Classic Alex 😛. You know what my comment is going to be.

Reply:
+1